compass_app/app/lib/features/results/business/model/destination.dart

```diff
@@ -0,0 +1,50 @@
+/// Model class for Destination data
+class Destination {
```

Comment:
I don't think it's strictly required to use code generation for data classes, but we should at least have a clear articulation of why we are not. For example, this `Destination` class lacking value equality could cause problems later on.

I am personally *quite* a fan of `pkg:freezed`, though that is just one of multiple possible choices.

Reply:
Agreed! Freezed could be a good option for this project, and we probably want to integrate it later on, as we add more features.

Reply:
This would be a good opportunity to 'be opinionated'. I agree that we probably want to add some sort of equality checking at the least and potentially code generation.  

@miquelbeltran this isn't something that we need to add in this PR necessarily, but good to consider for later.